### ISSUE TITLE
Check model 1 for bugs

### ISSUE THREAD

**Issue by ils246** [ils246/SCM#10]
When you have the time, please take a look at the new code for model1. 

It no longer takes a list of pvalues, but rather two values - pStar and pHubs. In the functions file you will find a new function called setHubs, that returns a list with all the pvalues for the agents. Just like  with the memory array, the index represents agents. 

I tried it with top =10 and top = 20 and the SCM is very similar or a little smaller than the one in model 0. In the experiments file you will find the code I used to run the model under 'Model 1' .

**Comment by crisjf** [comment]
I simplified the code for model 0 and model 1.

I left the two old functions after model0_old and model1_old. If you find the changes agreeable, please delete these two old functions.

**Comment by ils246** [comment]
Looks like you might have missed something. I think you want ps instead of p in line 98 in models.py? Otherwise how would you pass the list of ps created by setHubs.

**Comment by crisjf** [comment]
I don't think it matters. _master() does not care whether if it gets a number or an iterable as the parameters for p, q, and beta.
